pkg/addons/data/CronJob.batch/openshift-etcd/etcd-backup-cleaner.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: etcd-backup-cleaner
+  namespace: openshift-etcd
+spec:
+  # todo: move to 1 day after testing
+  schedule: "0 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: etcd-operator
```

Comment:
The etcd-operator has elevated access where this cronjob only needs to get and delete an etcdbackup. Needs its own service account, role, and rolebinding.